src/main/java/org/folio/oaipmh/dao/impl/SetDaoImpl.java

```diff
@@ -138,7 +150,7 @@ private SetRecord toDatabaseSetRecord(Set set) {
     if (isNotEmpty(set.getName())) {
       dbRecord.setName(set.getName());
     }
-    if (isNotEmpty(set.getDescription())) {
+    if (Objects.nonNull(set.getDescription())) {
```

Comment:
Does this mean that there's no way to remove a description?

Reply:
> Does this mean that there's no way to remove a description?

No, at opposite its a fix for what you've said) before we hadn't the possibility to remove description.

Reply:
if I start with:

```
{
  "id": "12345",
  "name": "foobar",
  "description": "hello"
}
```

and put:
```
{
  "id": "12345",
  "name": "foobar"
}
```

Then description will be null, the condition will evaluate to false and the description will remain as `"hello"`, right?  I guess I'm not sure why you need these conditionals at all...

Reply:
> if I start with:
> 
> ```
> {
>   "id": "12345",
>   "name": "foobar",
>   "description": "hello"
> }
> ```
> 
> and put:
> 
> ```
> {
>   "id": "12345",
>   "name": "foobar"
> }
> ```
> 
> Then description will be null, the condition will evaluate to false and the description will remain as `"hello"`, right? I guess I'm not sure why you need these conditionals at all...

Yes, i got it what you mean, i think you are right here. I've removed it already. Thank you.